recipe/build.sh

```diff
@@ -31,9 +31,4 @@ $PYTHON setup.py \
         install --single-version-externally-managed \
                 --record=record.txt
 
-#cp ${SRC_DIR}/python/build/dist/lib/*${SHLIB_EXT} ${PREFIX}/lib
-cp -r ${SRC_DIR}/python/build/dist/lib/* ${PREFIX}/lib
```

Comment:
The builds are now failing with (e.g. for `linux-64`):
`['  ERROR (pyarrow,lib/python3.10/site-packages/pyarrow/libarrow_python_flight.so): lib/python3.9/site-packages/pyarrow/libarrow_python.so not found in any packages']`
At a first glance, could be related, maybe the libraries need to be copied from a different location?

Reply:
The build worked locally until I was told about the `--error-overlinking` flag that Prefect runs with. Maybe that's why I didn't have those directories locally... Let me put those lines back in and give that a try.

Reply:
[Here's](https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/Package_building/More_debugging/Overlinking_overdepending_dso_whitelist.md#overlinking) an explanation of overlinking, if it helps

Reply:
Eric figured it out but I don't think what he did is mentioned in the docs. I'll have to sync with him latter on why his fixed worked, and hopefully I'll remember to add the details somewhere later.